docs/root/intro/arch_overview/operations/draining.rst

```diff
@@ -12,6 +12,8 @@ various events. Draining occurs at the following times:
 * The server is being :ref:`hot restarted <arch_overview_hot_restart>`.
 * Individual listeners are being modified or removed via :ref:`LDS
   <arch_overview_dynamic_config_lds>`.
+* The server begins shutdown sequence via the :ref:`drain_listeners?graceful
```

Comment:
Line 39 below needs updating. Please take a general pass through this page. You have as good an understanding of this as any, and we expect contributors to care for the docs and improve them as they make changes. Thanks!

Reply:
My bad, I'm kind of rushing for a deadline and writing docs in markdown was a little confusing. Reorganized a lot of this doc and added more detail.